apis/frontend/implementation-guide-session-api/advanced-usage/spa-out-out-of-session-tracking.md

```diff
@@ -46,3 +46,13 @@ Having done so, you can also verify that you have opted out by using the `isDoNo
 nostojs(api => console.log(api.visit.isDoNotTrack()));
 ```
 
+## Impact on features
+
+This section lists the impact of enabled `doNoTrack` mode on various Nosto products
```

Comment:
consider using "various Nosto features" or something similar instead as "Nosto products" could be misinterpreted